xbmc/video/VideoInfoScanner.cpp

```diff
@@ -1491,41 +1519,44 @@ namespace VIDEO
     CGUIListItem::ArtMap art = pItem->GetArt();
```

Comment:
Starting `GetArtwork` with
```
   int artworkLevel = CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
      CSettings::SETTING_VIDEOLIBRARY_ARTWORK_LEVEL);
    if (artworkLevel == CSettings::VIDEOLIBRARY_ARTWORK_LEVEL_NONE)
      return;
```
will skip fetching actors thumbs when art level = none and also the other unnecessary processing.
(Can't insert it as a comment on the line I want, this is near as can get)